Driverless AI/ES/introducción-a-machine-learning-automático-con-driverless-ai/introducción-a-machine-learning-automático-con-driverless-ai.md

```diff
@@ -200,83 +201,84 @@ En el conjunto de datos encontrarás 1,309 filas, una fila por cada pasajero, y
 |boat|Número de bote| - |
 |body|Numero de cuerpo| - |
 |home.des|Destino| - |
-
+ 
 ### Agregar los datos
-
+ 
 1\. Presiona en **Add a Dataset(or Drag and Drop) (Agregar un conjunto de datos (o arrastrar y soltar))**
-
+ 
 2\. Selecciona **FILE SYSTEM (SISTEMA DE ARCHIVO)**
-
+ 
 ![add-dataset-file-system](assets/add-dataset-file-system.jpg)
-
+ 
 3\. Ingresa los siguiente 
 1. /data/TestDrive/titanic.csv en la barra de búsqueda.
 2. Selecciona titanic.csv y después titanic.csv 
 3. Luego haga clic aquí: **Click to Import Selection.**
-
+ 
 ![select-titanic-dataset](assets/select-titanic-dataset.jpg)
-
+ 
 4\. Si el archivo se carga con éxito deberás ver lo siguiente:
-
+ 
 ![titanic-set-overview](assets/titanic-set-overview.jpg)
-
+ 
 **Notas**
-
+ 
 1. Podrás ver en cada columna:
-
+ 
     - **File name:** Nombre del archivo del conjunto de datos 
     - **File Path:** Ruta del archivo 
     - **File Size:** Tamaño del archivo 
     - **Rows:** Número de filas Rows
     - **Columns:** Número de columnas 
     - **Status:** Estado del archivo Status
-
+ 
 2. Opción para regresar a la página anterior
-
+ 
 ### Más información y recursos
-
-[Aprende más sobre los tipo de formatos para archivos de conjunto de datos que se pueden cargar](http://docs.h2o.ai/driverless-ai/latest-stable/docs/userguide/datasets.html#adding-datasets)
-
+ 
+[Aprende más sobre los tipos de formatos para archivos de conjunto de datos que se pueden cargar](http://docs.h2o.ai/driverless-ai/latest-stable/docs/userguide/datasets.html#adding-datasets)
+ 
 Para descubrir más conjunto de datos, revisa [Conjunto de datos Kaggle](https://www.kaggle.com/datasets)
 
-## Tarea 4: Explora los Detalles de Datos y AutoViz
 
+## Tarea 4: Explora los Detalles de Datos y AutoViz
+ 
 ### Detalles
-
+ 
 Ahora vamos a explorar el conjunto de datos Titanic que acabamos de cargar.
-
+ 
 1\. Continuando en la **Dataset Overview page** (página de descripción general del conjunto de datos), haga clic en el conjunto de datos titanic.csv. Aparecerán las siguientes opciones:
-
+ 
 ![titanic-set-actions](assets/titanic-set-actions.jpg)
-
+ 
 - Details (Detalles) - Vea un resumen del conjunto de datos o obtenga una vista previa del conjunto de datos
 - Visualize (Visualizar) - Visualice el conjunto de datos con gráficos disponibles
 - Split (Dividir) - Dividir el conjunto de datos
 - Predict (Predecir) - Ejecuta un experimento usando Driverless AI (IA sin conductor)
```

Comment:
I don't think you need to translate Driverless AI because this is the name of a product. If you do decide to translate it, please do so the first time you use "Driverless AI" in the tutorial